tensorflow/tools/api/golden/tensorflow.train.-looper-thread.pbtxt

```diff
@@ -36,7 +36,7 @@ tf_class {
   }
   member_method {
     name: "join"
-    argspec: "args=[\'self\', \'timeout\'], varargs=None, keywords=None, defaults=[\'None\'], "
+    argspec: "args=[\'self\', \'timeout\', \'balancing\'], varargs=None, keywords=None, defaults=[\'None\', \'True\'], "
```

Comment:
@annarev @martinwicke Hi, I hide `ApplyAdaMax` op, however API compatibility check still failed. And I run
```bash
    $ bazel build tensorflow/tools/api/tests:api_compatibility_test
    $ bazel-bin/tensorflow/tools/api/tests/api_compatibility_test \
          --update_goldens True
```
I found the failure seems unrelated.

Reply:
This diff is comparing "join" method in LooperThread class with a golden file. LooperThread inherits from Python's threading.Thread (https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/training/coordinator.py). So, the "join" method here is just "join" in threading.Thread.

We really shouldn't be comparing arguments of a Python method though (that is, this shouldn't be a part of our golden file). I will look into fixing it. 

Can you revert changes to tensorflow.train.-looper-thread.pbtxt for now? I can re-trigger the builds once you do. Also, which Python version are you using?

Reply:
Thanks for your help, @annarev . Sorry that I drop the commit and rebase the branch before I realize that git-revert is a better choice.
In a word, I have drop the change as requested, could you help restart all tests? Thanks.